sdk/storage/Azure.Storage.Blobs/src/Azure.Storage.Blobs.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyTitle>Microsoft Azure.Storage.Blobs client library</AssemblyTitle>
+    <AssemblyName>Azure.Storage.Blobs</AssemblyName>
```

Comment:
AssemblyName and RootNamespace are redundant and not necessary.

Reply:
Fixed